configuration/coding_plans.py

```diff
@@ -228,6 +228,22 @@ def get_survey_coding_plans(pipeline_name):
                        )
                    ],
                    ws_code=CodeSchemes.WS_CORRECT_DATASET.get_code_with_match_value("household language"),
+                   raw_field_fold_strategy=FoldStrategies.assert_equal),
+
+        CodingPlan(raw_field="government_priority_raw",
+                   time_field="government_priority_time",
+                   coda_filename="government_priority.json",
+                   coding_configurations=[
+                       CodingConfiguration(
+                           coding_mode=CodingModes.SINGLE,
+                           code_scheme=CodeSchemes.GOVERNMENT_PRIORITY,
+                           cleaner=None,
+                           coded_field="government_priority_coded",
+                           analysis_file_key="government_priority",
+                           fold_strategy=FoldStrategies.assert_label_ids_equal
```

Comment:
the follow up might have more than one label per message.

Reply:
Usually CSAP projects only take one label in the follow-ups. But good point, that may not be true here given the question. I'll check with the team, but address in a future PR if needed because it will likely break automated analysis, and I don't want to hold up push to Coda.